test/UserRepository-test.js

```diff
@@ -2,7 +2,63 @@ import { expect } from 'chai';
 import UserRepository from '../src/UserRepository';
```

Comment:
I just saw that you and Joan used different `import` syntax. I believe we want to go with yours since it is what is asking us to do in the project spec.

Reply:
Ah ha! Now I get it, Ricardo. I will update my two files accordingly and repush my branch for the User!